web/js/my-services.js

```diff
@@ -0,0 +1,11 @@
+$(document).ready(function() {
+
+    // To support non-js visitors, hide service details if JavaScript is enabled
```

Comment:
wait wut? I get what you mean, but without context this makes little sense :wink: